rpmlint/checks/FilesCheck.py

```diff
@@ -554,6 +555,8 @@ def check(self, pkg):
                 self.output.add_info('W', pkg, 'manifest-in-perl-module', f)
             elif f == '/usr/info/dir' or f == '/usr/share/info/dir':
                 self.output.add_info('E', pkg, 'info-dir-file', f)
+            elif makefile_regex.search(f):
+                self.output.add_info('E', pkg, 'makefile-junk', f)
```

Comment:
I'm not sure this makes sense as an error. For example, the SELinux policy package ships Makefiles as `-devel` content, and there are Makefile-based build systems and tools (blech). I'd downgrade this to a warning.

Reply:
Does the selinux put it in some determined location? If by design we could whitelist it and error out on everything else it is probably better.

Reply:
There are various Makefiles in `/usr/share/selinux/devel` and subdirectories.

Reply:
So just excluding /usr/share/selinux would be golden right?

Reply:
Probably.

Reply:
Added there, now just the tests.